src/components/shared/VideoWithPlaceholder.tsx

```diff
@@ -18,20 +18,18 @@ import {
 } from "react-navigation";
 import Video from "react-native-video";
 
-interface OwnProps {
+type Props = {
   uri: string;
 }
 
-type Props = OwnProps & NavigationInjectedProps;
-
 const initialState = {
   isPressed: false,
   videoLoaded: false,
   videoPaused: true,
 };
 
 export class VideoWithPlaceholderView extends React.Component<
-  Props,
+  Props & NavigationInjectedProps,
```

Comment:
What was wrong with what you were doing before?

Reply:
Just wanted to be consistent wherever we use `withNavigation`/`NavigationInjectedProps`. Initially I created an extra type everywhere PropsWithNavigation but seemed overly verbose